### PR TITLE
Allow participant summary api to query results using offset field

### PR DESCRIPTION
**Pull request by shyamnaru**
This will optimize WorkQueue pagination in HealthPro.

Ex: When HealthPro sends _offset = 60 & _count = 10 the RDR then returns 10 results starting from 60.

cc: @jt2k

### CODE REVIEW

**Comment by danqrodney** [comment]
Looks fine, please add tests here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/test/unit_test/api_test/participant_summary_api_test.py

**Comment by shyamnaru** [comment]
Sure, will add some tests.

**Comment by shyamnaru** [comment]
@danqrodney FYI: This PR is ready for review

File: rest-api/test/unit_test/api_test/participant_summary_api_test.py [hunk 1]
```diff
@@ -512,6 +512,50 @@ def test_get_summary_list_returns_total(self):
...
+        'education': PMI_SKIP_CODE,
+        'income': PMI_SKIP_CODE,
+        'dateOfBirth': datetime.date(1978, 10, 9),
+        'CABoRSignature': 'signature.pdf',
+      }
+      self.post_demographics_questionnaire(participant_id, questionnaire_id, **answers)
+
+    response = self.send_get('ParticipantSummary?_count=10')
+    # Pass offset query parameter
+    response2 = self.send_get('ParticipantSummary?_count=5&_offset=2')
```
Comment (danqrodney) [inline rest-api/test/unit_test/api_test/participant_summary_api_test.py]: Please add tests that show:

1) What happens if you request more participants than remain (e.g. offset = 6, count = 5)
2) What happens if you set an offset bigger than the number of participants (e.g. offset = 10)

File: rest-api/test/unit_test/api_test/participant_summary_api_test.py [hunk 2]
```diff
@@ -512,6 +512,50 @@ def test_get_summary_list_returns_total(self):
...
+
+    # Verify offset results
+    # Prove that first and last entry in response2 matches 3rd and 7th entry in response
+    self.assertEqual(response2['entry'][0], response['entry'][2])
```
Comment (danqrodney) [inline rest-api/test/unit_test/api_test/participant_summary_api_test.py]: Please check that you got back exactly 5 participants in response2.

File: rest-api/test/unit_test/api_test/participant_summary_api_test.py (hunk 1 above)
Comment (shyamnaru) [reply rest-api/test/unit_test/api_test/participant_summary_api_test.py]: Sure, will add these test cases.

File: rest-api/test/unit_test/api_test/participant_summary_api_test.py (hunk 2 above)
Comment (shyamnaru) [reply rest-api/test/unit_test/api_test/participant_summary_api_test.py]: Okay

**Pull request merged**